views/trust-information.html

```diff
@@ -0,0 +1,105 @@
+                        {{ boi.first_name }} {{ boi.last_name }}
+                      </label>
+                    </div>
+                  {% endif %}
+                {% endfor %}
+              {% endif %}
+              {% if (beneficial_owners_corporate and beneficial_owners_corporate.length > 0) %}
+                {% for boc in beneficial_owners_corporate %}
+                  {% if (boc.trustees_nature_of_control_types and boc.trustees_nature_of_control_types.length > 0) %}
+                    <div class="govuk-checkboxes__item">
+                      <input class="govuk-checkboxes__input" id="{{ boc.id }}" name="boCheckbox" type="checkbox" value="{{ boc.id }}">
+                      <label class="govuk-label govuk-checkboxes__label" for="{{ boc.id }}">
+                        {{ boc.name }}
+                      </label>
+                    </div>
+                  {% endif %}
+                {% endfor %}
+              {% endif %}
+            </div>
+          </fieldset>
+        </div>
+
+        <h2 class="govuk-heading-m">Enter trust information from the Excel document</h2>
+        <div class="govuk-form-group">
+          {{ govukTextarea({
+            name: "more-detail",
+            id: "more-detail",
+            label: {
+              text: "If you've already completed the trust Excel document, follow the instructions in the document to paste the data into the box:",
+              classes: "govuk-body",
+              isPageHeading: false
+            }
+          }) }}
+        </div>
+        <details class="govuk-details" data-module="govuk-details">
+          <summary class="govuk-details__summary">
+            <span class="govuk-details__summary-text">
+              I have not completed the trust Excel document yet
+            </span>
+          </summary>
+          <div class="govuk-details__text" id="more-detail" value="data.['more-detail']">
+            <p class="govuk-body">Enter the trust information into the <a href="//{{CDN_HOST}}/static/services/overseas-entities/OverseasEntityTrustDataCapture.xlsm" class="govuk-link">Trust Excel document (52KB)</a>. Be aware that this service will time out if you do not use it for 60 minutes. Your answers will not be saved.</p>
+          </div>
+        </details>
+
+        {{ govukInsetText({
+          html: '
+            <h2 class="govuk-heading-m">Information shown on the public register</h2>
+            <p>None of the information you provide about the trust will be shown on the public Register of Overseas Entities. However, it may be shared with HMRC.</p>
+          '
+        }) }}
+        {{ govukButton({
+          text: "Add another trust",
+          href: "/register-an-overseas-entity/trust-information"
+        }) }}
+
+        <p>
+          {{ govukButton({
+            text: "No more to add",
+            href: "/register-an-overseas-entity/check-your-answers",
```

Comment:
could you add an 'id' attribute of 'submit' to the 'no more to add' button? (example here - https://github.com/companieshouse/overseas-entities-web/blob/main/views/beneficial-owner-type.html#L102)
It will allow our auto tests in the pipeline to pick up the button on the screen.